.github/markets/pr_template.md

```diff
@@ -95,4 +95,11 @@ Each json file under the [configs](../../configs) folder correspond to their res
 |`show_from`   |`string`   |false   |The date and time when the market banner is scheduled to begin displaying. |If not provided, the banner will be shown immediately.<br /><br /> This field **MUST** follow the valid ISO 8601 format <br /> e.g. *2024-01-23T09:00+00:00* (23 Jan 2024, 9am UTC) |
 |`show_until`   |`string`   |false   |The date and time when the market banner is scheduled to stop displaying. |If not provided, the banner will continue to display indefinitely.<br /><br /> This field **MUST** follow the valid ISO 8601 format <br /> e.g. *2024-01-23T09:00+00:00* (23 Jan 2024, 9am UTC) | 
 |`content`   |`string`   |true   |The content shown on the market banner. | 
-|`hideable`   |`boolean`   |false   |Indicates if user can hide the banner by clicking on the close button |If set to `false`, the close button will not be rendered on the banner, and user will not be able to dismiss the banner. |
\ No newline at end of file
+|`hideable`   |`boolean`   |false   |Indicates if user can hide the banner by clicking on the close button |If set to `false`, the close button will not be rendered on the banner, and user will not be able to dismiss the banner. |
+
+## Native Contract Info
+
+| Field                        | Type                     | Required | Description                                                                                                          | Notes |
+| ---------------------------- | ------------------------ | -------- | -------------------------------------------------------------------------------------------------------------------- | ----- |
+| `native_token_contracts_map` | `NativeTokenContractMap` | false    | Here is a list of denoms that support deposits with native tokens, including the corresponding native token address. |       |
```

Comment:
1. Replace `NativeTokenContractMap` => `obj` so that the type is clearer.
2. Change Description text to `Map of token denoms to their respective contract addresses on the native chain`